### PR TITLE
Prepare for 0.8.0 release

### PR DESCRIPTION
**Pull request by StephenWeatherford**
- [x] CHANGELOG.md
- [x] README.md
- [x] Update 3rd party notices

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,49 @@
 # Change Log
 All notable changes to the "vscode-azurearmtools" extension will be documented in this file.
 
-## Version 0.8.0 (2019-10-01)
+## Coming Soon
+- Support for variable iteration ("COPY blocks")
```
Comment (jimmylewis) [inline CHANGELOG.md]: I've never seen a "coming soon" section in a changelog before...

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,49 @@
...
+- Language server using lots of memory [#324](https://github.com/microsoft/vscode-azurearmtools/issues/324)
+- Colorization of parameters and variables should be case-insensitive [#298](https://github.com/microsoft/vscode-azurearmtools/pull/298)
+- Improved description for uniqueString [#309](https://github.com/microsoft/vscode-azurearmtools/issues/309)
+- Updated dotnet core install scripts to latest versions from https://dot.net/v1/dotnet-install.sh and https://dot.net/v1/dotnet-install.ps1
+- Lookup of object variable property names is now correctly case-insensitive
+- JSON keys such as "Parameters" are now correctly case-insensitive
+- Azure Resource Manager Template Server crashed on MacOS [#332](https://github.com/microsoft/vscode-azurearmtools/issues/332) - ~/.local/share folder will now be created if necessary
+
+### Changed
+- Added "(ARM)" to extension name (to "Azure Resource Manager (ARM) Tools" from
```
Comment (jimmylewis) [inline CHANGELOG.md]: nit: the ( before the sub-clause isn't closed

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,49 @@
...
+  - Parameter count validation
+  - Rename user function parameters
+  - Hover information
+  - Signature help
+- Added basic snippet support. This makes Sam Cogen's [Azure Resource Manager Snippets](https://marketplace.visualstudio.com/items?itemName=samcogan.arm-snippets) extension no longer necessary when using this extension.  If you have snippet suggestions, you can add them to our [repo](https://github.com/microsoft/vscode-azurearmtools/issues).
+- Hover information for JSON properties (outside of expressions)
+- Fewer false positives
+  - We now allow string values to be automatically coerced to non-string types as appropriate (e.g., "false" can be used in place of false), as the ARM backend allows
+  - We now allow non-string values to be automatically coerced to string as appropriate (e.g. false can be used in place of "false")
+
```
Comment (jimmylewis) [inline CHANGELOG.md]: - Added broader support for ARM deployment scopes (resource group, subscription, management group, tenant).  That was new for some of our features at least.

File: CHANGELOG.md (hunk 1 above)
Comment (StephenWeatherford) [reply CHANGELOG.md]: Me neither :-), but AlexF was talking about putting roadmaps there.  Was going to see what he thought.

File: CHANGELOG.md (hunk 3 above)
Comment (StephenWeatherford) [reply CHANGELOG.md]: Unfortunately the change on Lidong's side to support these had to be reverted, so we get validation errors if these are used.  Will add in 0.8.1.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,49 @@
...
-## Version 0.8.0 (2019-10-01)
+## Coming Soon
+- Support for variable iteration ("COPY blocks")
+
+## Version 0.8.0 (2019-10-28)
+### Added
+- Greatly improved schema errors and completion
+  - Our new language server now has a better understanding of Azure Resource Manager templates and can therefore provide a better error and completion experience beyond that provided using standard JSON validation
+  - It now narrows down its schema validation based on the resource type and apiVersion properties of each resource.
+  - No longer suggests changing the resource type if the apiVersion is not supported, or if a property is missing or invalid
```
Comment (bhsubra) [inline CHANGELOG.md]: I think we can remove this. My change to remove missing 'apiVersion' if apiProfile and type are specified (and if we are able to get apiVersion from cache) is not in yet.

@jimmylewis , do you agree?

File: README.md [hunk 5]
```diff
@@ -8,7 +8,15 @@ This extension provides language support for Azure Resource Manager deployment t
 - Provides a language server that understands Azure Resource Manager deployment template files
 - ARM Template Outline view for easy navigation through large templates
 - Colorization for Template Language Expressions (TLE)
-- Analyze and validate JSON syntax, JSON schema conformance for Azure resources, string expressions and deployment issues
+- Analyze and validate JSON syntax, JSON schema conformance for Azure resources, string expressions issues that would affect deployment
+  - **NEW in 0.8.0!** Our new language server now has a better understanding of Azure Resource Manager templates and can therefore provide a better error and completion experience beyond that provided using standard JSON validation
+    - It narrows down its schema validation based on the resource type and apiVersion properties of each resource.
+    - No longer suggests changing the resource type if the apiVersion is not supported, or if a property is missing or invalid (as tools using standard JSON validation would do)
```
Comment (bhsubra) [inline README.md]: Same as above comment

File: CHANGELOG.md (hunk 4 above)
Comment (jimmylewis) [reply CHANGELOG.md]: I think this is from the ambiguous resolution issue we used to have.  If two schemas matched the other properties equally poorly, we'd flag apiVersion or resourceType as incorrect (even though they were valid).  Now with the new evaluation, we validate those first, then only flag any other properties that remain incorrect with respect to these two.

I think this is different from the apiProfile fix that you have in progress.

Comment (bhsubra) [reply CHANGELOG.md]: Thanks for the clarification Jimmy!

File: README.md (hunk 5 above)
Comment (bhsubra) [reply README.md]: Resolving this based on previous conversation

**Comment by alex-frankel** [comment]
What a great release!!

**Pull request merged**